content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -2,22 +2,22 @@
 This is an article about Sockets. It will give you a glimpse of how clients and servers communicate over the internet.
 
 Table of content:
-- [What is a socket](#what-is-a-socket)
-- [Programming on Server-side](#programming-on-server-side)
-- [](#)
-- [](#)
-- [](#)
+- [socket definition](#socket-definition)
+- [Server-Side Programming](#server-side-programming)
+- [Client-Side Programming](#client-side-programming)
+- [How Run The Programs on different applications](#how-to-run-the-programs-on-diffrent-applications)
+- [Conclusion](#conclusion)
 
 ### Introduction
-Socket programming is a way of connecting two computers on a network to allow the exchange of information between the two machines that are running over `JRE`(Java Runtime Environment). The connection can be through **connection-oriented protocol** or **connectionless** but for our case, we will be using connection-oriented which is TCP/IP.
+Socket programming is a way of connecting two computers over a network to allow the exchange of information.The connection can be through **connection-oriented protocol** or **connectionless**. In our case, we will be using  TCP/IP a connection-oriented protocol.
```

Comment:
Change: 

> Socket programming is a way of connecting two computers over a network to allow the exchange of information.The connection can be through **connection-oriented protocol** or **connectionless**. In our case, we will be using TCP/IP a connection-oriented protocol.

To:

**Socket programming is a way of connecting two computers over a network to allow the exchange of information. The connection can be through **connection-oriented protocol** or **connectionless**. In our case, we will be using TCP/IP a connection-oriented protocol.**

You were just missing a space here: `information.The connection` but otherwise, good job!